.github/workflows/agw-docker-load-test.yaml

```diff
@@ -1,7 +1,8 @@
 ---
 name: AGW docker load test
 
-on:  # yamllint disable-line rule:truthy
+on:
+  # yamllint disable-line rule:truthy
```

Comment:
The disable-line statement is vestigial since adding `truthy: disable` to config/yamllint_config.yml. Whether it's on the same line has no impact.

Instead, I propose to delete all these statements, since they reduce readability.